crates/iota-sdk/src/wallet_context.rs

```diff
@@ -25,6 +25,8 @@ use tracing::warn;
 
 use crate::{IotaClient, iota_client_config::IotaClientConfig};
 
+/// Wallet for managing accounts, objects, and interact with client APIs.
+// Mainly used in the CLI and tests.
```

Comment:
Was this meant to not be doc comment?

Reply:
Yes, do you think it should be a doc comment?

Reply:
No it's fine, just checking